tests/Mindee.IntegrationTests/TestingUtilities.cs

```diff
@@ -0,0 +1,33 @@
+namespace DefaultNamespace
```

Comment:
```suggestion
namespace Mindee.IntegrationTests
```
?

Reply:
Default IDE suggestion that I forgot to change :skull: